ios/Widget-Bridging-Header.h

```diff
@@ -0,0 +1 @@
+#import <react-native-palette-full/RNPalette.h>
```

Comment:
where do we use this?

Reply:
Leftover from when I was previously using an obj c pod. removed